### PR TITLE
added pretty-ms to tick-sync-tracker.js

### PR DESCRIPTION
**Pull request by JeremiGendron**
Issue #688 - looking for feedback as my testing environment is not proper atm. The package is included properly and should work, but again my environment is limited  I would like someone to validate it gets printed to the logger properly.

## Proposed changes

As @faustbrian suggested, pretty-ms package does provide an easy to use function for printing human-readable time. I invoke its function and indicate it to follow the format: Xd Yh Zm As and pass the refactored variable tracker.msLeft in the logger string to be displayed while syncing.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (improve a current implementation without adding a new feature or fixing a bug)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Build (changes that affect the build system)
- [ ] Docs (documentation only changes)
- [ ] Test (adding missing tests or fixing existing tests)
- [ ] Other... Please describe:

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/docs/contributing) documentation
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

looking for responses

### CODE REVIEW

**Comment by faustbrian** [comment]
Thanks for the PR, will test it in a few.

File: packages/core-blockchain/package.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
     "async": "^2.6.1",
     "delay": "^2.0.0",
     "interval-to-human": "^0.1.1",
-    "xstate": "^3.2.1"
+    "xstate": "^3.2.1",
+    "pretty-ms": "^3.2.0"
```
Comment (faustbrian) [inline packages/core-blockchain/package.json]: This is not properly installed. It needs to be removed and then added with `lerna add pretty-ms --scope=@arkecosystem/core-blockchain`.

File: packages/core-blockchain/lib/utils/tick-sync-tracker.js [hunk 2]
```diff
@@ -2,6 +2,8 @@ const container = require('@arkecosystem/core-container')
 const logger = container.resolvePlugin('logger')
 const database = container.resolvePlugin('database')
 
+const prettyMs = require('pretty-ms')
```
Comment (faustbrian) [inline packages/core-blockchain/lib/utils/tick-sync-tracker.js]: This should be the first line of the file as it is a third-party dependency.

File: packages/core-blockchain/lib/utils/tick-sync-tracker.js [hunk 3]
```diff
@@ -12,22 +14,22 @@ module.exports = async block => {
...
       networkHeight: container.resolvePlugin('p2p').getMonitor().getNetworkHeight(),
       downloadedBlocks: height,
-      blockPerSecond: 0,
+      blockPerMs: 0,
       percent: 0,
       secondsLeft: 0
     }
   }
 
   tracker.downloadedBlocks += 400
```
Comment (faustbrian) [inline packages/core-blockchain/lib/utils/tick-sync-tracker.js]: This was changed to `block.data.height` in a recent commit, your fork must be old. You should merge master into your branch.

File: packages/core-blockchain/lib/utils/tick-sync-tracker.js [hunk 4]
```diff
@@ -12,22 +14,22 @@ module.exports = async block => {
...
-  tracker.secondsLeft = (tracker.networkHeight - tracker.downloadedBlocks) / tracker.blockPerSecond
+  tracker.blockPerMs = ((new Date().getTime()) - tracker.start) / tracker.downloadedBlocks
+  tracker.msLeft = (tracker.networkHeight - tracker.downloadedBlocks) / tracker.blockPerMs
 
   if (tracker.percent < 100) {
     const downloadedBlocks = tracker.downloadedBlocks.toLocaleString()
     const networkHeight = tracker.networkHeight.toLocaleString()
 
-    logger.printTracker('Fast Sync', tracker.percent, 100, `(${downloadedBlocks} of ${networkHeight} blocks)`)
+    logger.printTracker('Fast Sync', tracker.percent, 100, `(${downloadedBlocks} of ${networkHeight} - ${prettyMs(tracker.msLeft, {secDecimalDigits: 0})})`)
```
Comment (faustbrian) [inline packages/core-blockchain/lib/utils/tick-sync-tracker.js]: The time seems to be quite a bit off. It starts at 9 hours and then jumps to 50 seconds.

**Comment by faustbrian** [comment]
@JeremiGendron I made you an example branch so you can compare with yours and see what the issues are https://github.com/ArkEcosystem/core/commit/c2371947012b66ec08d63c1fbd4fe7374617ee6d.

File: packages/core-blockchain/package.json (hunk 1 above)
Comment (JeremiGendron) [reply packages/core-blockchain/package.json]: ty for this insight

**Comment by JeremiGendron** [comment]
close for #691 and check it out if time allows it

**Pull request closed without merging**